demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/config/general.json

```diff
@@ -14,7 +14,6 @@
   "StartWithBlankPage": "true",
   "NavigationBar": {
     "LogoImageURL": "img/logo.png",
-    "BackgroundColor": "#F57636",

```

Comment:
If we're losing the possibility of customization with this specific change, we might have to reconsider or investigate other possible solutions. I'd let the style tag as an exception for this specific case, unless @nbellocam has a nicer solution.

Reply:
I thought several solutions but they were more complex than leaving this style as an exception. I restored this attribute to the config file